sdgym/summary.py

```diff
@@ -95,46 +91,49 @@ def summarize(data, baselines=(), datasets=None):
         pandas.DataFrame
     """
     if datasets is not None:
-        data = data[data.dataset.isin(datasets)]
+        data = data[data.Dataset.isin(datasets)]
 
-    baselines_data = data[data.synthesizer.isin(baselines)]
-    data = data[~data.synthesizer.isin(baselines)]
-    no_identity = data[data.synthesizer != 'DataIdentity']
+    baselines_data = data[data.Synthesizer.isin(baselines)]
+    data = data[~data.Synthesizer.isin(baselines)]
+    no_identity = data[data.Synthesizer != 'DataIdentity']
 
     coverage_perc, coverage_str = _coverage(data)
-    solved = data.groupby('synthesizer').apply(lambda x: x.normalized_score.notnull().sum())
+    solved = data.groupby('Synthesizer').apply(lambda x: x.Quality_Score.notnull().sum())
 
     results = {
-        'total': len(data.dataset.unique()),
+        'total': len(data.Dataset.unique()),
         'solved': solved,
         'coverage': coverage_str,
         'coverage_perc': coverage_perc,
         'time': _seconds(data),
-        'best': _best(no_identity, 1, 'normalized_score', False),
+        'best': _best(no_identity, 1, 'Quality_Score', False),
         'avg score': _mean_score(data),
-        'best_time': _best(no_identity, 1, 'model_time', True),
-        'second_best_time': _best(no_identity, 2, 'model_time', True),
-        'third_best_time': _best(no_identity, 3, 'model_time', True),
+        'best_time': _best(no_identity, 1, 'Train_Time', True),
+        'second_best_time': _best(no_identity, 2, 'Train_Time', True),
+        'third_best_time': _best(no_identity, 3, 'Train_Time', True),
```

Comment:
@npatki Are these still the desired columns to get in the output or are some of these unnecessary now?

Reply:
@amontanez24 we haven't really discussed what the final output summary should be. In any case, it would change because the underlying benchmarking method has also changed. So it seems fine to get rid of.